src/generated/TixiHelper.cpp

```diff
@@ -459,6 +459,7 @@ namespace tigl
             TixiSaveElementInternal(tixiHandle, xpath, tixiUpdateTextElement, tixiAddTextElement, value);
         }
 
+        // TODO: updating an empty text doesn't work : no error, but the text is not updated
```

Comment:
I quickly checked our code bases and notices, that this line is in the generator code since the creation of the file IOHelper.cpp on 2016-08-09. I further noticed, that you have removed this line on your ms/new_cs_math branch. I guess the change was overwritten now, after I have regenerated the code.
I'll guess this is no longer a problem than and remove the line from the generator code